src/wp-includes/class-wp-theme-json.php

```diff
@@ -679,7 +679,6 @@ public static function get_element_class_name( $element ) {
 		array( 'spacing', 'margin' ),
 		array( 'spacing', 'padding' ),
 		array( 'typography', 'lineHeight' ),
-		array( 'shadow', 'defaultPresets' ),
```

Comment:
The `* @since 6.5.0 Added `background.backgroundSize`, `dimensions.aspectRatio`, and `shadow.defaultPresets`.` line above needs to be updated. Basically reverting https://core.trac.wordpress.org/changeset/57717

Reply:
Good catch! Fixed in c35f2ddb5dab7b8c72d115858eafc49b700ad040.